### PR TITLE
Sql db2 pullas

### PR DESCRIPTION
**Pull request by Justuxs**

### CODE REVIEW

File: CityPuzzle/Login And SingUp/SignUpPage.xaml.cs [hunk 1]
```diff
@@ -47,9 +47,9 @@ public static void Validation(List<Entry> fields)
                         if (!Valid_Username.IsMatch(field.Text)) throw new BadInputdException("nuo 6 iki 12 ilgio", field.Placeholder, field);
                         else if (!User.CheckUser(field.Text)) throw new BadInputdException("Naudotojas su tokiu prisijungimo vardu užimtas.Pakeiskite prisijungimo vardą ir bandykite dar kartą.", field);
                         break;
-                    case "Slaptazodis":
-                        if (!Valid_Password.IsMatch(field.Text)) throw new BadInputdException("Password must be atleast 8 to 15 characters and contains atleast one Upper case and numbers.", field);
-                        break;
+                    //case "Slaptazodis":
+                       // if (!Valid_Password.IsMatch(field.Text)) throw new BadInputdException("Password must be atleast 8 to 15 characters and contains atleast one Upper case and numbers.", field);
```
Comment (AllBill127) [inline CityPuzzle/Login And SingUp/SignUpPage.xaml.cs:51]: Why comment out Password validation?

File: CityPuzzle/Classes/Sql.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@ namespace CityPuzzle.Classes
 {
     class Sql
     {
-        public static string ConnStr = "Server = tcp:citypuzzledb.database.windows.net,1433;Initial Catalog = citypuzzledb; Persist Security Info=False;User ID = citypuzzleuser; Password=User123*; MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout = 30;";
+ m        public static string ConnStr = "Server=tcp:citypuzzle.database.windows.net,1433;Initial Catalog=CityPuzzle;Persist Security Info=False;User ID=citypuzzle;Password=User123*;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;";
```
Comment (AllBill127) [inline CityPuzzle/Classes/Sql.cs:15]: m letter is an error. Must delete

**Comment by AllBill127** [comment]
Observation not connected to commits on this branch: User class QuestsCompleted variable is not instantiated which is bad and causes an error when registering new user. This should be an important fix but i will make a pull request with SignUpPage updates into here so I will add a fix then

File: CityPuzzle/Login And SingUp/SignUpPage.xaml.cs (hunk 1 above)
Comment (Justuxs) [reply CityPuzzle/Login And SingUp/SignUpPage.xaml.cs:51]: I wanted to create a short password to log in faster :) . Sorry, I will fix it.

**Comment by AllBill127** [comment]
After many messy tries we correctly updated and merged all branches into this one and got rid of merge conflicts

**Comment by AllBill127** [comment]
No more updates should be and will be done in this branch. We can merge and delete it

**Pull request merged**